dcc-storage-server/pom.xml

```diff
@@ -156,6 +156,21 @@ ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF S
           </execution>
         </executions>
       </plugin>
+      
+      <!-- Jar -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifest>
+              <mainClass>org.icgc.dcc.storage.server.ServerMain</mainClass>

```

Comment:
it's already been removed - when I was trying to get it to work, I thought I needed it.